src/test/features/controls-spec.js

```diff
@@ -158,7 +158,7 @@ suite('ModelViewerElementBase with ControlsMixin', () => {
 
         test('updates with current orbit after interaction', async () => {
           const controls = element[$controls];
-          controls.adjustOrbit(0, 0.5, 0);
+          controls.adjustOrbitFromUserInteraction(0, 0.5, 0);
```

Comment:
I renamed this method because it seemed that currently this function was only being used on user interaction via mouse/touch.

By doing this I could emit distinct events for when the user was interacting vs just orbit changing.

Reply:
Yes I think so. Let's say you're updating the orbit of the model via `<model-viewer>` attributes. eg let's say an intro animation. Once this intro animation has finished likely you'd like it to begin to auto rotate immediately.

To me pausing auto rotate after user interaction is a specialized case and should be the only time autorotate is paused. Specifically in commerce you might be looking at the details of the item you're thinking of purchasing and you likely don't want the model to begin to auto rotate as soon as you let go.

Although it could be argued that `autoRotate` should simply not activate anymore after user interaction. 🤷‍♂️ Maybe some UX folks could way in on this eg @bldelgado

Reply:
Fair enough, it sounds like the distinction is useful 👍

Reply:
I'd definitely parrot what mikko said about having the ability to pause the autorotate, especially if the model is zoomed in. We tested out a model with auto rotate while zoomed and when looking at details, it was sometimes jarring when there was a lot of negative space. Our example was using sunglasses and tall lamps. Some products lend themselves better to being zoomed out and then zoomed back in when the detail is on a thin piece of geometry.